### PR TITLE
Task sf/j238 private subscription functionality

### PR DESCRIPTION
**Pull request by Sayonarko**

### CODE REVIEW

**Comment by Sayonarko** [comment]
[Private subscription functionality](https://trello.com/c/xdeWfBpz/238-private-subscription-functionality)

**Review by JuliaDzyuba — DISMISSED** [review]

File: frontend/src/helpers/uuid/get-uuid/get-uuid.helper.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { v4 as uuid } from 'uuid';
+
+const getUuid = (): string => {
```
Comment (what1s1ove) [inline frontend/src/helpers/uuid/get-uuid/get-uuid.helper.ts]:
```suggestion
const getRandomId = (): string => {
```
don't forget to rename the file
and move it to the `string` helpers

File: frontend/src/components/podcast-invite/podcast-invite.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { useParams, useDispatch, useEffect } from 'hooks/hooks';
+import { Loader } from 'components/common/common';
+import { podcast as podcastActions } from 'store/actions';
+import { PageParams } from './common/types/types';
+
+const PodcastInvite: React.FC = () => {
+  const dispatch = useDispatch();
+  const { code } = useParams<PageParams>();
+
```
Comment (what1s1ove) [inline frontend/src/components/podcast-invite/podcast-invite.tsx:9]: if we don't have an user we should redirect to the `/login page`

**Review by what1s1ove — CHANGES_REQUESTED** [review]
Good! 👍

File: frontend/src/components/podcast-invite/podcast-invite.tsx (hunk 2 above)
Comment (Sayonarko) [reply frontend/src/components/podcast-invite/podcast-invite.tsx:9]: `/podcasts/invite/:code` this is a private route

Comment (what1s1ove) [reply frontend/src/components/podcast-invite/podcast-invite.tsx:9]: Good!

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**